data/sql/updates/pending_db_world/wip_ahnkahet_azc.sql

```diff
@@ -0,0 +1,102 @@
+-- Change this back to INSERT once rewrite is complete
+REPLACE INTO `version_db_world` (`sql_rev`) VALUES
+('1599510903096685100');
```

Comment:
you don't have to do this manually. Just run:

```
./data/sql/updates/pending_db_world/create_sql.sh
```

this will create a new file for you, just copy paste your SQL there.

See step 5. of https://www.azerothcore.org/wiki/How-to-create-a-PR

Reply:
Sql will get reworked before final PR stage ;)